doc/dev/release.rst

```diff
@@ -216,7 +214,7 @@ so:
         virtualenv build
         cd build
         source bin/activate
-        pip install -U setuptools==3.4.1 wheel
+        pip install wheel
```

Comment:
Are you sure? So many issues are covered up or emerge from having very out of date or very recent setuptools...